aws/src/main/java/org/apache/iceberg/aws/glue/IcebergToGlueConverter.java

```diff
@@ -132,4 +138,61 @@ static void validateTableIdentifier(TableIdentifier tableIdentifier) {
     validateNamespace(tableIdentifier.namespace());
     validateTableName(tableIdentifier.name());
   }
+
+  static List<Column> toColumns(Schema schema) {
```

Comment:
Normally, we'd put conversion methods like this in a class called `GlueSchemaUtil` instead of this converter class that is meant to be internal. It would not be unreasonable to open up these methods later and if we do we want some consistency in the public API.

Reply:
Yes I agree, I am aware of the `HiveSchemaUtil` class, but given these are just for informational purpose, because people who have access to their Glue table might not have permission in S3, but they should still see information like schema, etc.

I don't want people to actually take a dependency or use it as some reference implementations. The actual source of truth is always the underlying Iceberg table metadata, that's why I am keeping them in this protected class. If in the future there is actually a way to perform some integration with services like Glue ETL, we will publish a more defined converter of schema that can be public, but so far there is no plan for it.

I have added some explanations in javadoc, please let me know if you are fine with it, thanks!